### PR TITLE
Sanity tweaks.

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
Reduces blood sanity damage from 0.5 to 0.25
Reduces remains sanity damage from 1 to 0.5
reduced human remains sanity damage from 5 to 2.5
Reduces passive insight gain from 0.05 to 0.025
Full Sanity raised from 150-300(This is what you start with.) Insight passive gain reduced from 0.5 to 0.25
insight gain multiplier lowered from 1.0 to 0.5
Low-threshold of environment damage raised from 20-100. suggestions were to raise this from 20-50 before sanity cap was doubled, that would be 33% of your max. To keep that percentage it was also doubled to maintain a nice level. This is only the 'lowest' that environmental sanity features can take you.

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by Guidesu** [comment]
i do not like it. Insight already takes a while to even gain, you are making it even more tedious to get a single level, it is already painfully slow even if you dungeon crawl on maints

File: code/modules/sanity/sanity_mob.dm [hunk 1]
```diff
@@ -53,13 +53,13 @@ GLOBAL_VAR_INIT(GLOBAL_INSIGHT_MOD, 1)
 	var/sanity_passive_gain_multiplier = 1
 	var/sanity_invulnerability = 0
 	var/level
-	var/max_level = 150 //Soj change to give a bit more breathing room
+	var/max_level = 300 //Soj change to make sanity less of a wacky rollercoaster.
```
Comment (cdb-is-not-good) [inline code/modules/sanity/sanity_mob.dm:56]:
```suggestion
	var/max_level = 200 //Soj change to make sanity less of a wacky rollercoaster.
```

File: code/modules/sanity/sanity_mob.dm [hunk 2]
```diff
@@ -75,7 +75,7 @@ GLOBAL_VAR_INIT(GLOBAL_INSIGHT_MOD, 1)
 	var/positive_prob_multiplier = 1
 	var/negative_prob = 30
 
-	var/view_damage_threshold = 20
+	var/view_damage_threshold = 100
```
Comment (cdb-is-not-good) [inline code/modules/sanity/sanity_mob.dm:78]:
```suggestion
	var/view_damage_threshold = 75
```

**Comment by Trilbyspaceclone** [comment]
i.g now i never can breakdown from anything it seems?
not even close well in combat or hanging around tons of dead roaches/bodies/ect

**Comment by benj8560** [comment]
I was having a real hard time getting my sanity to drop to. Was standing beside a mountain of gibbs and it barely budged

File: code/modules/sanity/sanity_mob.dm [hunk 3]
```diff
@@ -53,13 +53,13 @@ GLOBAL_VAR_INIT(GLOBAL_INSIGHT_MOD, 1)
...
 	var/level
-	var/max_level = 150 //Soj change to give a bit more breathing room
+	var/max_level = 200 //Soj change to make sanity less of a wacky rollercoaster.
 	var/level_change = 0
 
 	var/insight
 	var/max_insight = INFINITY
-	var/insight_passive_gain_multiplier = 0.5
-	var/insight_gain_multiplier = 1
+	var/insight_passive_gain_multiplier = 0.25
```
Comment (cdb-is-not-good) [inline code/modules/sanity/sanity_mob.dm:61]:
```suggestion
	var/insight_passive_gain_multiplier = 0.5
```

File: code/modules/sanity/sanity_mob.dm [hunk 4]
```diff
@@ -53,13 +53,13 @@ GLOBAL_VAR_INIT(GLOBAL_INSIGHT_MOD, 1)
...
+	var/insight_gain_multiplier = 0.5
```
Comment (cdb-is-not-good) [inline code/modules/sanity/sanity_mob.dm:62]:
```suggestion
	var/insight_gain_multiplier = 1
```

File: code/modules/sanity/sanity_mob.dm [hunk 5]
```diff
@@ -75,7 +75,7 @@ GLOBAL_VAR_INIT(GLOBAL_INSIGHT_MOD, 1)
 	var/positive_prob_multiplier = 1
 	var/negative_prob = 30
 
-	var/view_damage_threshold = 20
+	var/view_damage_threshold = 75
```
Comment (cdb-is-not-good) [inline code/modules/sanity/sanity_mob.dm:78]:
```suggestion
	var/view_damage_threshold = 35
```

File: code/modules/sanity/sanity_mob.dm [hunk 6]
```diff
@@ -31,7 +31,7 @@ GLOBAL_VAR_INIT(GLOBAL_INSIGHT_MOD, 1)
 
 #define SANITY_CHANGE_FADEOFF(level_change) (level_change * 0.75)
 
-#define INSIGHT_PASSIVE_GAIN 0.05
+#define INSIGHT_PASSIVE_GAIN 0.025
```
Comment (cdb-is-not-good) [inline code/modules/sanity/sanity_mob.dm:34]:
```suggestion
#define INSIGHT_PASSIVE_GAIN 0.05
```

**Pull request merged**